ballerina/modules/parser/utils.bal

```diff
@@ -80,3 +80,7 @@ isolated function isValidFirstChar(string char) returns boolean = @java:Method {
 isolated function isValidChar(string char) returns boolean = @java:Method {
     'class: "io.ballerina.stdlib.graphql.runtime.parser.ParserUtils"
 } external;
+
+public isolated function getHashCode(object {} obj) returns string = @java:Method {
```

Comment:
Let's create an issue to track the progress of replacing this function with an ID field.

Reply:
Issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/3881